### PR TITLE
[FIX] Resolves https://github.com/odoo/odoo/issues/6216

### PR DESCRIPTION
**Pull request by daramousk**

### CODE REVIEW

**Comment by NL66278** [comment]
@daramousk Goodmorning George. Amazing if the solution to this longstanding problem in the end is so simple. But the best place to put your PR is on the original odoo/odoo repository. Maybe, who knows, Odoo SA will accept it. And also place the PR on the OCA/OCB backports. One off the agreements in the oca is that all PR's for oca/ocb need first to be done on the offical Odoo repositories. Except for Odoo versions that are no longer officially supported.

File: doc/cla/individual/daramousk.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+I hereby agree to the terms of the Odoo Individual Contributor License
+Agreement v1.0.
+
+I declare that I am authorized and able to make this agreement and sign this
+declaration.
+
+Signed,
+
+George Daramouskas gdaramouskas@therp.nl https://github.com/daramousk
```
Comment (hbrunn) [inline doc/cla/individual/daramousk.md:11]: @NL66278 I think we better put Georgios into our company CLA. As you did all the commits there up to now, I think it speeds up stuff if you add him too there: https://github.com/odoo/odoo/blame/8.0/doc/cla/corporate/therp.md
@daramousk you've read Ronald's post about this not being the good place for you patch, right?

Comment (hbrunn) [reply doc/cla/individual/daramousk.md:11]: oh, and while being at it, we should add him to the OCA cla too

Comment (daramousk) [reply doc/cla/individual/daramousk.md:11]: @hbrunn Yes, I have opened https://github.com/odoo/odoo/pull/17852.

I am closing this one.

Comment (hbrunn) [reply doc/cla/individual/daramousk.md:11]: dont forget to cherry-pick your commit to a PR in https://github.com/OCA/OCB, where you link the PR to odoo (background is that we ourselves use OCB, and the OCA's rule is that only PRs are merged that have a PR into Odoo proper)

**Pull request closed without merging**